cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-databases-show-databases, SHOW DATABASES>>                                                                                                                       | Database administration       | Returns information about all databases.
+| <<administration-databases-show-databases, SHOW DEFAULT DATABASE>>                                                                                                                | Database administration       | Returns information about the default database.
+| <<administration-databases-create-database, CREATE [OR REPLACE] DATABASE ... [IF NOT EXISTS]>>                                                                                    | Database administration       | Crates a new database.
+| <<administration-databases-drop-database, DROP DATABASE ... [IF EXISTS]>>                                                                                                         | Database administration       | Deletes a specified database.
+| <<administration-databases-start-database, START DATABASE>>                                                                                                                       | Database administration       | Starts up a specified database.
+| <<administration-databases-stop-database, STOP DATABASE>>                                                                                                                         | Database administration       | Stops a specified database.
+| <<administration-security-roles-show, SHOW [ALL \| POPULATED] ROLES [WITH USERS]>>                                                                                                | User and role administration  | Returns information about all or populated roles, optionally including the assigned users.
+| <<administration-security-roles-create, CREATE [OR REPLACE] ROLE ... [IF NOT EXISTS] ... [AS COPY OF]>>                                                                           | User and role administration  | Creates new roles.
+| <<administration-security-roles-drop, DROP ROLE [IF EXISTS]>>                                                                                                                     | User and role administration  | Deletes a specified role.
+| <<administration-security-users-show, SHOW USERS>>                                                                                                                                | User and role administration  | Returns information about all users.
+| <<administration-security-users-create, CREATE [OR REPLACE] USER ... [IF NOT EXISTS] SET PASSWORD ... [SET PASSWORD CHANGE [NOT] REQUIRED] [SET STATUS [ACTIVE \| SUSPENDED]]>>   | User and role administration  | Creates a new user and sets the password for the new account. Optionally the account status can also be set and if the user should change the password upon first login.
```

Comment:
One more part of this command is optional, the second SET PASSWORD, and one is not, ACTIVE|SUSPENDED:
`CREATE [OR REPLACE] USER ... [IF NOT EXISTS] SET PASSWORD ... [[SET PASSWORD] CHANGE [NOT] REQUIRED] [SET STATUS {ACTIVE \| SUSPENDED}]`